style.css

```diff
@@ -437,6 +437,139 @@ input.form-control::placeholder {
   margin: 15px auto 0 auto;
 }
 
+/* Mobile Modals windows */
+.modelContainer {
+  display: none;
+  flex-direction: column;
+  background: var(--white-color);
+  border-radius: 20px;
+  margin: 30px auto 25px auto;
+  width: 95%;
+  height: 620px;
+  padding: 10px;
+  position: fixed;
+  animation: smoth-effet-open 0.7s;
+  top: -10px;
+  left: 0;
+  right: 0;
+  bottom: 10px;
+  object-fit: contain;
+}
+
+@keyframes smoth-effet-open {
+  from {
+    transform: scale(0);
+  }
+
+  to {
+    transform: scale(1);
+  }
+}
+
+.modal .short_desc {
+  display: flex;
+  align-items: center;
+  margin: 3px 0;
+}
+
+.modelContainer.show {
+  display: flex;
+  z-index: 100;
+}
+
+.closeModal {
+  animation: smoth-effet-close 0.7s;
+}
+
+@keyframes smoth-effet-close {
+  from {
+    transform: scale(0);
+  }
+
+  to {
+    transform: scale(1);
+  }
+}
+
+.modal__headMain {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: -15px 0 -12px 0;
+}
```

Comment:
- [ ] [REQUIRED] You have done a great job implementing some of the recommended changes, however, you still need to  fix the `see live` button positioning because it overflows from the modal container to match the Figma design. The `See source` button doe not have much space in between it and the `See live` button as you have provided. Kindly ensure that you stick to the Figma design as possible.
### Your Design
![Screenshot from 2022-04-05 19-12-18](https://user-images.githubusercontent.com/58692590/161823231-e0d61cf8-d16b-4da0-b64e-8b01977e343e.png)

### FIgma Template
![Screenshot from 2022-04-05 19-15-36](https://user-images.githubusercontent.com/58692590/161823335-f3a54b3e-a115-4f80-838a-9990406c5da1.png)